etc/kayobe/ansible/deploy-radosgw-usage-exporter.yml

```diff
@@ -0,0 +1,152 @@
+      command: >
+        {{ venv }}/bin/openstack
+        endpoint list --service object-store --interface internal
+        --format json
+      environment: "{{ openstack_auth_env }}"
+      delegate_to: "{{ groups['controllers'][0] }}"
+      vars:
+        ansible_host: "{{ hostvars[groups['controllers'][0]].ansible_host }}"
+      register: endpoint
+      run_once: true
+      when: stackhpc_enable_radosgw_usage_exporter
+
+    - name: Ensure radosgw_usage_exporter container is running
+      community.docker.docker_container:
+        name: radosgw_usage_exporter
+        image: ghcr.io/stackhpc/radosgw_usage_exporter:0.1.0
+        network_mode: host
+        env:
+          RADOSGW_SERVER: "{{ radosgw_server }}"
+          ADMIN_ENTRY: admin
+          ACCESS_KEY: "{{ ec2.Access }}"
+          SECRET_KEY: "{{ ec2.Secret }}"
+          VIRTUAL_PORT: "{{ stackhpc_radosgw_usage_exporter_port | string }}"
+        entrypoint: "{{ [ 'python', '-u', './radosgw_usage_exporter.py', '--insecure' ] if not stackhpc_radosgw_usage_exporter_verify else omit }}"
+      vars:
+        ec2: "{{ credential.stdout | from_json | first }}"
+        host: "{{ endpoint.stdout | from_json | first }}"
+        radosgw_server: "{{ host.URL | regex_replace('(https?://)([^:/]+):([0-9]+)/.*', '\\1\\2:\\3') }}" # Drop Swift api address
+      become: true
+      when: stackhpc_enable_radosgw_usage_exporter
+
+    - name: Create radosgw-usage-exporter directory
+      ansible.builtin.file:
+        path: /opt/kayobe/radosgw-usage-exporter/
+        state: directory
+      when:
+        - stackhpc_enable_radosgw_usage_exporter
+        - stackhpc_radosgw_usage_exporter_cacert | length > 0
+
+    - name: Copy CA certificate to RADOS gateway usage exporter nodes
+      ansible.builtin.copy:
+        src: "{{ stackhpc_radosgw_usage_exporter_cacert }}"
+        dest: "/opt/kayobe/radosgw-usage-exporter/{{ stackhpc_radosgw_usage_exporter_cacert | basename }}"
+      register: copy_to_node_result
+      when:
+        - stackhpc_enable_radosgw_usage_exporter
+        - stackhpc_radosgw_usage_exporter_cacert | length > 0
+
+    - name: Copy CA certificate to RADOS gateway usage exporter container
+      community.docker.docker_container_copy_into:
+        container: radosgw_usage_exporter
+        path: "{{ copy_to_node_result.dest }}"
+        container_path: "/usr/local/share/ca-certificates/{{ copy_to_node_result.dest | basename }}"
+      become: true
+      when:
+        - stackhpc_enable_radosgw_usage_exporter
+        - stackhpc_radosgw_usage_exporter_cacert | length > 0
+
+    - name: Update CA certificate of RADOS gateway usage exporter container
+      community.docker.docker_container_exec:
+        container: radosgw_usage_exporter
+        command: update-ca-certificates
+        user: root
+      become: true
+      when:
+        - stackhpc_enable_radosgw_usage_exporter
+        - stackhpc_radosgw_usage_exporter_cacert | length > 0
```

Comment:
Again, add the `stackhpc_radosgw_usage_exporter_cacert | length > 0` condition to a block if you can